docs/data/material/getting-started/support/support.md

```diff
@@ -61,12 +61,13 @@ The open-source community is always welcome to submit new features and bug fixes
 
 The current status of each MUI version is as follows:
 
-- MUI Core v5 (emotion): ✅ In active development.
-- MUI Core v4 (hooks): ⚠️ Only handle security fixes.
-- MUI Core v3 (change supported browsers): ❌ Inactive.
+- MUI Core v5: **✅ In active development**.
```

Comment:
Maybe a table link in https://mui.com/versions/#release-schedule could work better by being easier to quickly scan. Not saying we need to go as deep as https://nodejs.org/en/about/releases/ but maybe there are stuff that are cheap and fast to copy for us and relevant.

Reply:
As this content is now living under the Versions page, the Release Schedule section is already there so it's easier to refer to.